project-bwk.yml

```diff
@@ -284,4 +285,17 @@ targets:
     dependencies:
       - target: Networking
       - package: SnapshotTesting
+  ViewInspectorTestHelpers:
+    type: framework
+    platform: iOS
+    settings:
+      base:
+        ENABLE_TESTING_SEARCH_PATHS: YES
+        GENERATE_INFOPLIST_FILE: YES
+    sources:
+      - path: ViewInspectorTestHelpers
+        excludes:
+          - "**/Info.plist"
```

Comment:
🤔 Should we generate the Info.plist and then exclude it? Would it be the same not to generate it?